File: java/CComponent.java
package cell;



public class CComponent implements ISerialize
{
	public int GetCComponentType() { return Df.CComponent.Null; }
	public String GetKey() { return new String(); }
	public CPacket Serialize() {return new CPacket();}
	public void Deserialize(String _str) {	}
}

File: js/CComponent.js



class CComponent extends ISerialize
{
	GetCComponentType() { return Df.CComponent.Null; }
	GetKey() { return ""; }
	Serialize()
	{
		var pac=new CPacket();
		pac.Push(this.GetCComponentType());
		return pac;
	}
	Deserialize(_str)
	{
		var pac=new CPacket();
		pac.Deserialize(_str);
	}
};

